wavefront/server/apps/floware/floware/server.py

```
@@ -261,6 +267,13 @@ async def lifespan(app: FastAPI):
             )
         )
 
+        # Start Redis Stream consumer for async execution status updates
+        async_agentic_exec_consumer = AsyncAgenticExecutionResultConsumer(
+            exec_repo=db_repo_container.async_agentic_execution_repository(),
+            cache_manager=db_repo_container.cache_manager(),
+        )
+        asyncio.create_task(async_agentic_exec_consumer.start())
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)